src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -549,29 +549,26 @@ private IEnumerable<TypeInferenceInfo> InferTypeInAttributeArgument(int index, I
                 return InferTypeInAttributeArgument(index, methods.Select(m => m.Parameters), argumentOpt);
             }
 
-            private IEnumerable<TypeInferenceInfo> InferTypeInArgument(int index, IEnumerable<IMethodSymbol> methods, ArgumentSyntax argumentOpt)
+            private IEnumerable<TypeInferenceInfo> InferTypeInArgument(int index, IEnumerable<IMethodSymbol> methods, ArgumentSyntax argumentOpt, InvocationExpressionSyntax parentInvocationExpressionOpt = null)
```

Comment:
can we make non-optional param and fixup callers to pass null if appropriate (or an actual value if not).

Reply:
@CyrusNajmabadi Great suggestion, done.